Week1/HW7.4.playground/Contents.swift

```diff
@@ -0,0 +1,66 @@
+/*
+ 1.1
+ 
+ 
+ variables can be changed constants are final
+ var myvar: Int = 2
+ no, it is a keyword designated for another purpose.
```

Comment:
In fact, the return keyword can be used in variables. Of course, In such cases, it is called by "a computed property" or "a getter for a variable". These computed properties is not used in storing but in computing and returning a values.

# Examples

```swift
var getString: String? {
    return "Value"
}
```

The above codes equals the following codes.

```swift
var getString: String? {
    get {
        return "Value"
    }
}
```

Reply:
No ones of the following codes returns itself.

```swift
var myValue: Int {
  get {
    return 10
  }
}
```

```swift
var sum: Int {
  lhs + rhs // return keyword can be skipped.
}
```